tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,472 @@ def test_reads_during_server_pod_restart(self):
+        thread_wri = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                      kwargs=args)
+        thread_wri.daemon = True  # Daemonize thread
+        thread_wri.start()
+        LOGGER.info("Step 3.1: Successfully started WRITEs with variable sizes objects"
+                    " in background")
+        LOGGER.info("Step 3.2: Start READs and verify DI on the written data in background")
+        output_rd = Queue()
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': test_prefix_read,
+                'nclients': 2, 'nsamples': 5, 'skipwrite': True, 'skipcleanup': True,
+                'output': output_rd, 'setup_s3bench': False}
+        thread_rd = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                     kwargs=args)
+        thread_rd.daemon = True  # Daemonize thread
+        thread_rd.start()
+        LOGGER.info("Step 3.2: Successfully started READs and verify on the written data in "
+                    "background")
+        LOGGER.info("Step 3: Successfully started READ/WRITEs/VERIFY with variable object sizes"
+                    " in background")
+        LOGGER.info("Waiting for %s seconds", HA_CFG["common_params"]["30sec_delay"])
+        time.sleep(HA_CFG["common_params"]["30sec_delay"])
+        LOGGER.info("Step 4: Restart the pod with replica method and check cluster status")
+        event.set()
+        resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                       restore_method=self.restore_method,
+                                       restore_params={"deployment_name": self.deployment_name,
+                                                       "deployment_backup":
+                                                           self.deployment_backup,
+                                                       "num_replica": self.num_replica,
+                                                       "set_name": self.set_name},
+                                       clstr_status=True)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+        self.restore_pod = False
+        LOGGER.info("Step 4: Successfully restart the pod with replica method and checked "
+                    "cluster status")
+        event.clear()
+        thread_wri.join()
+        thread_rd.join()
+        LOGGER.info("Step 5.1: Verify status for In-flight WRITEs while %s server pod "
+                    "restarted ", pod_name)
+        responses_wr = dict()
+        while len(responses_wr) != 2:
+            responses_wr = output_wr.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_wr["pass_res"])
+        fail_logs = list(x[1] for x in responses_wr["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
```

Comment:
can we combine pass_logs and fail_logs instead and then check for failures as we are not expecting any failures?
cc: @ApurwaMohite

Reply:
Yes, we can combine. But while logging error we need to mention both the lists separately.